public/app/features/logs/components/LogRows.tsx

```diff
@@ -105,7 +105,7 @@ class UnThemedLogRows extends PureComponent<Props, State> {
   };
 
   popoverMenuSupported() {
-    if (!config.featureToggles.logRowsPopoverMenu || this.props.app !== CoreApp.Explore) {
+    if (!config.featureToggles.logRowsPopoverMenu) {
```

Comment:
Are there any other apps that shouldn't get this functionality? Will every app with a `onClickFilterOutString` or `onClickFilterString` support the popover menu or degrade gracefully?

Reply:
No. The only place where these props were defined was Explore, and the only data sources that could use it were those which either implemented `hasToggleableQueryFiltersSupport` or `modifyQuery`. This check never really needed to exist.